package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.15.3",
     "github": "^9.2.0",
     "mz": "2.6.0",
```

Comment:
Mostly because that would imply having to use `try`/`catch` blocks to manage exceptions properly.

Also, taking into account that we were already using promises in this file, it makes sense to me to keep the same structure.